ofrak_patch_maker/ofrak_patch_maker/toolchain/gnu.py

```diff
@@ -46,7 +46,6 @@ def __init__(
                 #      if sections contain more than one function =/
                 "-fno-merge-constants",  # avoids sections like .rodata.cst16, .rodata.str1.1 etc
                 "-fno-reorder-functions",
-                "-Wall",
```

Comment:
What is the reason for this change? I don't see any additional warnings generated by the big endian test cases you added. If there's a specific spurious warning you ran into consistently, it should be disabled by a conditional `-Wno` in `gnu_arm.py`

Reply:
I just removed one because the `-Wall` option was present twice in this list